resource/command.go

```diff
@@ -48,14 +49,15 @@ func (c *Command) GetExec() string {
 }
 
 func (c *Command) Validate(sys *system.System) []TestResult {
+	ctx := context.WithValue(context.Background(), "id", c.ID())
```

Comment:
Using context values for this is ok, but probably more idiomatic these days to use a structured logger like the (new) slog or something like logrus and then pass that down with key-vals added.

Reply:
I'm very interested to see what ends up as idiomatic with the introduction of slog.

Seems like it's been a heavily debated topic: https://github.com/golang/go/issues/58243

I'm okay with pivoting to whatever becomes the norm once the dust settles a bit on slog and patterns emerge.